content/en/docs/components/pipelines/sdk/v2/component-development.md

```diff
@@ -75,7 +75,7 @@ When you design your component's code, consider the following:
 
     _Parameters_ typically represent settings that affect the behavior of your pipeline.
     Parameters are passed into your component by value, and can be of any of
-    the following types: `int`, `double`, `float`, or `str`. Since parameters are
+    the following types: `int`, `float`, `str`, `bool`, `dict`, `list`. Since parameters are
```

Comment:
Forgot to ask, is there a KFP SDK version requirement for this change?

Reply:
This is available in 1.6.1 -- the very first stable release we made for Vertex AI preview.
But good point, I do see this doc still mentions 1.6.0rc0. Will update that version.

Reply:
Please make the following change to this list. (Otherwise, LGTM.)

`dict`, `list` -> `dict`, or `list`

Reply:
Done.